src/logger.rs

```diff
@@ -130,17 +130,27 @@ impl Logger {
 ///
 /// Configuration of the logger can also be controlled via environment variables:
 /// * AMETHYST_LOG_STDOUT - determines the output to the terminal
+///     * "no" / "off" disables logging to stdout
+///     * "plain" / "yes" enables logging to stdout
+///     * "colored" enables logging and makes it colored
 /// * AMETHYST_LOG_LEVEL_FILTER - sets the log level
+///     * "off" disables all logging
+///     * "error" enables only error logging
+///     * "warn" only errors and warnings are emitted
+///     * "info" only error, warning and info messages
```

Comment:
When I tried info in my projects, info + warn would actually NOT show. I'm not sure why that was the case.
Using debug would show info + warn + errors though.

Reply:
I tried every possible filter, all worked as expected.